packages/rspeedy/plugin-react-alias/src/index.ts

```diff
@@ -1,10 +1,12 @@
 // Copyright 2024 The Lynx Authors. All rights reserved.
 // Licensed under the Apache License Version 2.0 that can be found in the
 // LICENSE file in the root directory of this source tree.
+import { readFileSync } from 'node:fs'
 import { createRequire } from 'node:module'
 import path from 'node:path'
 
 import type { RsbuildPlugin } from '@rsbuild/core'
+import semver from 'semver'
```

Comment:
Avoid importing all the `semver` package. Use `semver/functions/gte.js` instead.

This would reduce the bundle size from 83.3 kB to 26.2 kB.